js/main.js

```diff
@@ -157,8 +158,12 @@ $cancelButton.addEventListener('click', function (event) {
 });
 
 $confirmButton.addEventListener('click', function (event) {
-  $targetedListItem.remove();
-  data.entries.splice(data.editing.entryId - 1, 1);
+  for (var entriesIndex = 0; entriesIndex < data.entries.length; entriesIndex++) {
+    if (currentEntry === data.entries[entriesIndex].entryId) {
```

Comment:
Instead of this `currentEntry` variable, I would just use `data.editing.entryId`